nnpdf_data/nnpdf_data/theory_cards/41041002.yaml

```
@@ -0,0 +1,42 @@
+ID: 41_041_002
+Comments: NNPDF4.1 NLO QCD QED
+PTO: 1
+FNS: FONLL-C
```

```suggestion
FNS: FONLL-B
```
eeehm, okay, I'll take the challenge to explain FONLL in brief :see_no_evil: 
1. the long explanation: read [our new FONLL paper](https://inspirehep.net/literature/2818030) and then take a look to Sec. 3.3 of [the original FONLL paper](https://arxiv.org/pdf/1001.2312)
2. the short explanation: FONLL combines fixed order ("FO") and resummed ("NLL") computations. The perturbative accuracy of the FO part is a parameter, as well as the logarithmic accuracy of the resummed part (so even though it is called "NLL" = next-to-leading log, we can in fact do it at any logarithmic order). The letter X in "FONLL-X" specifies those two accuracies.
   - for NLO PDF fits we could use "FONLL-A", but in practice we use "FONLL-B"
   - for NNLO PDF fits we always use "FONLL-C"
   - for N3LO PDF fits we use "FONLL-E" - remember #2385

---

Thank you so much!